arm32: Build fails with error: 'hasBaseLine8MPreReqs' was not declared in this scope

After fixing #481, the build on arm32 still fails with:

```
g++  -fno-PIE -c  -I../../gcc/../zlib -g -O2 -DIN_GCC     -fno-exceptions -fno-rtti -fasynchronous-unwind-tables -W -Wall -Wno-narrowing -Wwrite-strings -Wcast-qual -Wno-error=format-diag -Wmissing-format-attribute -Woverloaded-virtual -pedantic -Wno-long-long -Wno-variadic-macros -Wno-overlength-strings -fno-common  -DHAVE_CONFIG_H -I. -I. -I../../gcc -I../../gcc/. -I../../gcc/../include -I../../gcc/../libcpp/include -I../../gcc/../libcody  -I../../gcc/../libdecnumber -I../../gcc/../libdecnumber/dpd -I../libdecnumber -I../../gcc/../libbacktrace   -o gcov.o -MT gcov.o -MMD -MP -MF ./.deps/gcov.TPo ../../gcc/gcov.c
../../gcc/config/arm/arm-rust.c: In function 'void arm_rust_target_cpu_info()':
../../gcc/config/arm/arm-rust.c:194:9: error: 'hasBaseLine8MPreReqs' was not declared in this scope; did you mean 'hasBaseline8MPreReqs'?
  194 |     if (hasBaseLine8MPreReqs && arm_arch_thumb2 && arm_arch7)
      |         ^~~~~~~~~~~~~~~~~~~~
      |         hasBaseline8MPreReqs
../../gcc/config/arm/arm-rust.c:204:9: error: 'arm_arch8_5' was not declared in this scope; did you mean 'arm_arch8_4'?
  204 |     if (arm_arch8_5)
      |         ^~~~~~~~~~~
      |         arm_arch8_4
../../gcc/config/arm/arm-rust.c:206:9: error: 'arm_arch8_6' was not declared in this scope; did you mean 'arm_arch8_4'?
  206 |     if (arm_arch8_6)
      |         ^~~~~~~~~~~
      |         arm_arch8_4
```

--- a/gcc/config/arm/arm-rust.c
+++ b/gcc/config/arm/arm-rust.c
@@ -191,7 +191,7 @@ void arm_rust_target_cpu_info(void) {
                                 && arm_arch_cmse && arm_arch_thumb1;
     if (hasBaseline8MPreReqs)
         rust_add_target_info("target_feature", "v8m");
-    if (hasBaseLine8MPreReqs && arm_arch_thumb2 && arm_arch7)
+    if (hasBaseline8MPreReqs && arm_arch_thumb2 && arm_arch7)
         rust_add_target_info("target_feature", "v8m.main");
     if (arm_arch8_1)
         rust_add_target_info("target_feature", "v8.1a");

--- a/gcc/config/arm/arm-rust.c
+++ b/gcc/config/arm/arm-rust.c
@@ -201,10 +201,6 @@ void arm_rust_target_cpu_info(void) {
         rust_add_target_info("target_feature", "v8.3a");
     if (arm_arch8_4)
         rust_add_target_info("target_feature", "v8.4a");
-    if (arm_arch8_5)
-        rust_add_target_info("target_feature", "v8.5a");
-    if (arm_arch8_6)
-        rust_add_target_info("target_feature", "v8.6a");
     if (bitmap_bit_p(arm_active_target.isa, isa_bit_armv8_1m_main))
         rust_add_target_info("target_feature", "v8.1m.main");
     if (bitmap_bit_p(arm_active_target.isa, isa_bit_mve)

